modules/update-cluster-version-object.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-cluster-version-object_{context}"]
+= The ClusterVersion object
+
+One of the resources that the Cluster Version Operator (CVO) monitors is `ClusterVersion`.
+
+`ClusterVersion` is a custom resource object that contains information relating to the cluster's version, such as the current and desired versions of the cluster.
+When the CVO observes that the desired version does not match the current version in the `ClusterVersion` resource, it attempts to initiate an update to reconcile the cluster with this new desired state.
```

Comment:
can we de-emphasize updates here with something like:

> The CVO continually reconciles the cluster with the target state declared in ClusterVersion `spec`.  When the desired release differs from the current one, that reconciliation updates the cluster.

to make it clear that "updating or not?" is a subset of the reconciliation the CVO is always doing?

Reply:
Does "the desired release" refer to anything beyond the desired version for the entire cluster? 

Also, if "that reconciliation updates the cluster" can refer to processes outside of the update process described in this "how updates work" assembly, I feel like using the word "update" to mean something other than "cluster-wise change from version X to version X+1" might be confusing to users. But I'm not sure.